shotgun_api3/shotgun.py

```diff
@@ -1519,7 +1534,7 @@ def get_attachment_download_url(self, attachment):
                 None, None, None))
         return url
 
-    def authenticate_human_user(self, user_login, user_password):
+    def authenticate_human_user(self, user_login, user_password, auth_token):
```

Comment:
Please also add the parameter to the doc string and ideally also information about which exceptions you can except to get back in various situations (wrong auth token, missing auth token, etc).